package.json

```
@@ -41,9 +42,9 @@
     "prettier": "^1.15.3"
   },
   "scripts": {
-    "dev": "next",
+    "dev": "sass --watch --no-source-map static:static & next",
     "build": "next build",
-    "start": "next start",
+    "start": "sass --no-source-map static:static && next start",
```

```suggestion
    "start": "next start",
```
I ended up suggesting a wrong change to this line. The SASS build must be done during the next build and not when running it. Forgiveness